### PR TITLE
Create Todo List App with Webpack

### PR DESCRIPTION
**Pull request by martinkarugaba**
# This is what I have worked on for this Pull Request

- [ ] Created an index.html file and wrote my HTML markup there. 
- [ ] Created an empty To Do List placeholder. The index.html file must be set as a template using the HTML Webpack Plugin.
- [ ] Created an index.js file and set an array of some simple to do tasks. 
- [ ] Each task object contains three keys: description, completed, index.
- [ ] Iterate over the tasks array and populate an HTML list item element for each task.
- [ ] On page load render the dynamically created list of tasks in the dedicated placeholder. 
- [ ] The list appears in order of the index values for each task.
- [ ] Created a style.css and set rules for the To Do List. 
- [ ] CSS is loaded by Webpack Style/CSS Loader. 
- [ ] My list is a clone of the part of the minimalist project that was given to us.
- [ ] All my src files are located in the `src` directory and are generated by webpack and served by webpack dev server from /dist folder.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  width: 3px;
+  background-color: #ccc;
+  border-radius: 50%;
+}
+
+.clear__btn {
+  padding: 0.5rem 1rem;
+  font-size: 1rem;
+  margin-top: 0.5rem;
+}
```
Comment (thecodechaser) [inline src/index.css:98]: - [ ] You did a good job on the styles but you need to work on the `Clear all completed` button. Please make sure the button has the same width as the tasks container and matches the given design.

### Your design

image

### Given design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @martinkarugaba👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,119 @@
...
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [About the Project] <a name="about-project"></a>
+
+**[awesome books]** is a project that displays a list of books.
```
Comment (Qoosim) [inline README.md:27]: - [OPTIONAL] I would recommend you to always make your about project to be inline with the title of your project. This milestone is `Todo-List`, project not `awesome books` project.

**Review by Qoosim — APPROVED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @martinkarugaba, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**